templates/list_pages/node--oe-list-page--full.html.twig

```diff
@@ -25,10 +25,10 @@
     <div class="col-12 col-lg-9">
       <div class="row">
         <div class="col-md-6 col-xl-8 align-self-center">
-          <h4 class="mb-0">
```

Comment:
it is ok to fix the tags, but we are breaking the page visually, we need to ensure that it looks as before